odkx-src/sync-endpoint.rst

```diff
@@ -132,3 +143,129 @@ Assigning users to groups
 
     a. Navigate to the :guilabel:`memberUid` section.
     b. Click modify group members to manage members.
+    
+.. _sync-endpoint-ldap-utils:
+
+Using LDAP utils
+"""""""""""""""""""""""""
+
+  The ldap-service container has ldap-utils installed. If you'd prefer, you may use that tool set to administer the LDAP directory as well. Use this command to
+  access them,
+ 
+    .. code-block:: console
+  
+     $ docker exec $(docker ps -f "label=com.docker.swarm.service.name=${STACK_NAME}_sync" --format '{{.ID}}') <LDAPTOOL> <ARGS>
+     
+    The docker exec takes a container name and a command . The command is executed in the specified container . 
+    
+.. _sync-endpoint-advanced:
+
+Advanced
+--------
+
+.. _sync-endpoint-ldap-defaults:
+
+Editing the defaults of LDAP directory
+"""""""""""""""""""""""""""""""""""""""""""""
+    Modify the ldap.env file to configure the environment variables . The default settings are as follows 
+    
+      .. code-block:: console
+      
+       # openldap
+       LDAP_ORGANISATION=Open Data Kit            // name of your organisation
+       LDAP_DOMAIN=example.org                    // domain of your organisation
+       LDAP_READONLY_USER=true                    // enable the read only user
+       LDAP_READONLY_USER_PASSWORD=readonly       // password for read only user
+       LDAP_ADMIN_PASSWORD=admin                  // default password for admin account 
```

Comment:
Could you add a note here that says if these options are modified, the corresponding options in Sync Endpoint's `security.properties` need to be modified to match.